apps/events/serializers/reservation_serializer.py

```diff
@@ -0,0 +1,20 @@
+from rest_framework import serializers
+
+from apps.events.models.reservation import Reservation
+
+
+class ReservationInSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Reservation
+        fields = (
+            "event",
+            "email",
+            "full_name",
+            "sex",
+        )
+
+
+class ReservationSerializer(serializers.ModelSerializer):
```

Comment:
Let use some conversion here. So instead of having two serializers which is better to declare **read only fields**